store/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonWriterBuilder.java

```diff
@@ -262,6 +271,18 @@ public CarbonWriterBuilder withLoadOption(String key, String value) {
     return this;
   }
 
+  /**
+   * To support the carbon table for sdk writer
+   *
+   * @param table carbon table
+   * @return CarbonWriterBuilder object
+   */
+  public CarbonWriterBuilder withTable(CarbonTable table) {
```

Comment:
There is a `CarbonWriterBuilder withSchemaFile(String schemaFilePath)` API already, is this `withTable` API required?

Reply:
resolved